packages/server/src/with-blitz.ts

```diff
@@ -18,6 +18,7 @@ export function withBlitz(nextConfig: any) {
       Object.assign({}, normalizedConfig, {
         experimental: {
           reactMode: "concurrent",
+          ...(nextConfig.experimental || {}),
```

Comment:
I didn't catch this earlier, but we need to use the normalized version. `nextConfig` can be a function.
```suggestion
          ...(normalizedConfig.experimental || {}),
```